tests/dummy/app/transitions.js

```diff
@@ -152,6 +152,14 @@ export default function(){
   );
 
   this.transition(
+    this.includingInitialRender(),
+    this.outletName('test'),
+    this.fromRoute('scenarios'),
```

Comment:
The test outlet never renders 'scenarios', so this constraint can never match.

You can leave the route constraints off here, because we only need to exercise the outletName constraint.